test/parallel/test-macos-app-sandbox.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const common = require('../common');
+if (process.platform !== 'darwin')
+  common.skip('App Sandbox is only avaliable on Darwin');
+
+const fixtures = require('../common/fixtures');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const child_process = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const os = require('os');
+
+const nodeBinary = path.resolve(
+  __dirname, '..', '..',
+  `out/${common.buildType}/node`);
```

Comment:
Any reason not to use `process.execPath`?

Reply:
Well it didn't occur to me. Updated as suggested.